### PR TITLE
Add test.txt file

### PR DESCRIPTION
**Pull request by JavadZandiyeh**
This is a new amazing feature.

### CODE REVIEW

**Comment by oibf** [comment]
Thank you for your contribution, but there are some fixes required to be considered. Okay?

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Adding new file
```
Comment (oibf) [inline test.txt:1]: Make some more details.

Comment (JavadZandiyeh) [reply test.txt:1]: I think no other details can be added here

File: test.txt
Comment (JavadZandiyeh) [inline test.txt:1]: greate

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# test
\ No newline at end of file
+# test readme
```
[the remark this answers is not in the document]
Comment (JavadZandiyeh) [reply README.md:1]: resolved

**Pull request merged**